public/src/main/java/com/revenuecat/purchases/DangerousSettings.kt

```diff
@@ -9,5 +9,18 @@ data class DangerousSettings(
      * automatically, and you will have to call syncPurchases whenever a new purchase is completed in order to send it
      * to the RevenueCat's backend. Auto syncing of purchases is enabled by default.
      */
-    val autoSyncPurchases: Boolean = true
-)
+    val autoSyncPurchases: Boolean = true,
+
+    /**
+     * Disable or enable offline entitlements. If this is disabled, RevenueCat will not try to provide entitlements in
+     * the unlikely case of an outage in our servers. This will prevent entitlements from being available for your
+     * users during an outage. This is enabled by default.
+     */
+    val offlineEntitlementsEnabled: Boolean = DEFAULT_OFFLINE_ENTITLEMENTS_ENABLED
```

Comment:
I thought we agreed that YAGNI?

Reply:
Ah, I misunderstood... I know we talked a bit about how users can potentially downgrade if they have issues with it, but I was still thinking this could be useful to make it easier to turn off since this feature is (hopefully) not going to be used in the short term and in case we run into issues in the future, users might have to downgrade too much. I guess releasing a new version disabling it could also be effective, but this might be easier... I'm ok if we don't want to overcomplicate the API though. CC @aboedo